javascript-create-module/template/package.json

```diff
@@ -17,42 +15,44 @@
     "format": "prettier --write --list-different .",
     "lint": "eslint .",
     "package": "yarn pack --out dist/package.tgz",
-    "watch": "vite build --watch",
+    "dev": "vite build --watch",
+    "watch": "run dev",
     "watch:callback": "run package && run deploy"
   },
   "dependencies": {
+    "canvas-confetti": "^1.9.3",
+    "clsx": "^2.1.1",
     "modern-normalize": "^3.0.1"
   },
   "devDependencies": {
     "@eslint-react/eslint-plugin": "^1.26.2",
     "@eslint/compat": "^1.2.6",
     "@eslint/js": "^9.20.0",
-    "@jahia/javascript-modules-library": "0.x || ^1.0.0",
-    "@jahia/vite-plugin": "^0.5.1",
+    "@jahia/javascript-modules-library": "latest",
```

Comment:
isn't it risky to go without SemVer control for our lib here?

Reply:
It is but that's the only way we have to have the template be up-to-date before we setup a more complex release flow. It's basically the same as `0.x || ^1.0.0` until v2